test/test_embeddings.py

```diff
@@ -115,6 +115,12 @@ def test_train_batch_KL(self):
 
         self._check_results(embed)
 
+    def test_model_update(self):
+        self.embed = sample_embed()
+        self.embed.update(5)
+        self.assertEquals(self.embed.mu.shape, (5, 2))
+        self.assertEquals(self.embed.sigma.shape, (5, 1))
```

Comment:
It would be nice to have a test here that the training processes did the right thing after adding ids to the embedding.  One idea:

* modify the training data piece of this test to make ids 8-9 correlated instead of 0-1.
* then split the training data into two groups: only rows excluding id 9 and rows only including id 9.
* create an embedding with 9 ids (0-8) and train on the first piece of data.
* then update it to add one ID (9)
* then train with the second bit of data
* then test that it recovers the expected nearest neighbors relationships (passes `_check_results`)